### PR TITLE
Modified submission test data for SubmissionSpec to live in spec rather

### PR DESCRIPTION
**Pull request by abaumann**
than OrientDbTestFixture, reused initialization code for consistency and
to ensure PoisonPill even if one test fails

### CODE REVIEW

File: src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphSubmissionDAOSpec.scala [hunk 1]
```diff
@@ -44,7 +44,7 @@ class GraphSubmissionDAOSpec extends FlatSpec with Matchers with OrientDbTestFix
     assert(!dao.list(workspace.namespace,workspace.name,txn).toSet.contains(submission3))
   }
 
-  it should "save, get, list, and delete two submission statuses" in withSubmissionData { txn =>
+  it should "save, get, list, and delete two sub}mission statuses" in withSubmissionData { txn =>
```
Comment (mattsooknah) [inline src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphSubmissionDAOSpec.scala]: typo?

**Comment by helgridly** [comment]
I'm okay with this but I'd like to be consistent. Can we have a rule that the test fixture defines the default test graph and anyone wanting custom things puts it in their own spec files?

**Comment by abaumann** [comment]
I'm fine with that rule, test fixture has default and empty

**Comment by helgridly** [comment]
Squash your commits and then :+1:

**Pull request merged**